### PR TITLE
fix the chrome shut down problem on changing theme

### PR DESCRIPTION
**Pull request by MislavPukljak**
fix of the problem "Chrome didn't shut down correctly".

### CODE REVIEW

File: bin/omakub-theme [hunk 1]
```diff
@@ -14,7 +14,9 @@ if [ -n "$THEME" ] && [ "$THEME" != "<<-back" ]; then
 
   # Chrome theme update
   if gum confirm "Do you want to update Chrome theme? If yes, Chrome will be restarted!"; then
-    pkill -9 chrome
+    pkill -f chrome
+    wait
+k
```
Comment (rmacklin) [inline bin/omakub-theme:19]: Typo?

**Comment by MislavPukljak** [comment]
Yes, thank you!

**Comment by dhh** [comment]
Hmm, I'm still seeing the same warning. "Didn't shut down correctly". It's not a blocker, but would be nice to get around it.

**Comment by MislavPukljak** [comment]
Alright, will look into it.

**Pull request merged**